docs/src/pages/[platform]/connected-components/account-settings/index.page.mdx

```diff
@@ -21,7 +23,39 @@ export async function getStaticProps() {
 
 <DeveloperPreview />
 
-## Account Settings
+Account Settings components are a set of standalone components that add [user management](https://docs.amplify.aws/lib/auth/manageusers/q/platform/js/) flows to your application with minimal boilerplate.
 
-TODO
+<Alert variation="warning">
+  Account Settings components require end user to be authenticated. You can use the [`Authenticator`](https://ui.docs.amplify.aws/react/connected-components/authenticator) component to add authentication UI to your application.
```

Comment:
what about using `connected-components/authenticator` since it's on the same domain

Reply:
You're right!
```suggestion
  Account Settings components require end user to be authenticated. You can use the [`Authenticator`](/connected-components/authenticator) component to add authentication UI to your application.
```